frameworks/hdfs/universe/marathon.json.mustache

```diff
@@ -40,6 +46,13 @@
     "LIBMESOS_URI": "{{resource.assets.uris.libmesos-bundle-tar-gz}}",
     "HDFS_URI": "{{resource.assets.uris.hdfs-tar-gz}}",
     "BOOTSTRAP_URI": "{{resource.assets.uris.bootstrap-zip}}",
+
+    {{#service.secret_name}}
+    "DCOS_SERVICE_ACCOUNT_CREDENTIAL": { "secret": "serviceCredential" },
+    "MESOS_MODULES": "{\"libraries\": [{\"file\": \"libdcos_security.so\", \"modules\": [{\"name\": \"com_mesosphere_dcos_ClassicRPCAuthenticatee\"}]}]}",
```

Comment:
The `MESOS_MODULES` env requires an absolute path of the form `file:///` and for those frameworks that untar the bundle in their MESOS_SANDBOX directory, we can't predict the absolute path.

For framework that untar the bundle at a known location, i.e., `/opt/libmesos-bundle`, we can include a `mesos_scheduler_authenticatee_module.json` file in the bundle and let the app point to it.